build.csx

```diff
@@ -89,3 +96,36 @@ public static void Cmd(string fileName, string args)
         }
     }
 }
+
+public static string ReadCmd(string fileName, string args)

```

Comment:
FYI I'm going to wrap both this and `Cmd()` into a NuGet package some time soon. Right now I'm repeating it in a few places.